pkg/expr/reader.go

```diff
@@ -74,6 +74,8 @@ func (h *ExpressionQueryReader) ReadQuery(
 		}
 		if err == nil && q.Settings != nil {
 			switch q.Settings.Mode {
+			case "":
```

Comment:
we could add a new constant ( `ReduceModeStric`) and refer it here, to make the case-structure more uniform, on the other hand maybe we want to emphasize that this is the `empty` case. WDYT?

Reply:
I like the idea of adding a new constant here personally, yes. I'd also like a comment, or maybe better some tests somewhere - describing that this can be present, or the whole section absent, for `Strict` reduce-mode to be applied.

Reply:
i added the const in https://github.com/grafana/grafana/pull/94838 , and the new unit test in https://github.com/grafana/grafana/pull/94837 will make it explicit that strict-mode and no-settings is the same (no mapper).